core/runtime/bindings.rs

```diff
@@ -456,6 +515,44 @@ fn op_ctx_function<'s>(
   v8fn
 }
 
+type AccessorStore<'a> = HashMap<String, (&'a OpCtx, Option<&'a OpCtx>)>;
+
+fn create_accessor_store(ctx: &OpMethodCtx) -> AccessorStore {
+  let mut store = AccessorStore::new();
+
+  for method in ctx.methods.iter() {
+    // Populate all setters first.
+    if method.decl.accessor_type == AccessorType::Setter {
+      let key = method.decl.name_fast.to_string();
+
+      // All setters must start with "__set_".
+      let key = key.strip_prefix("__set_").expect("Invalid setter name");
```

Comment:
Is there a particular reason for this? Why don't getters require to start with `__get_`?

Reply:
This is to allow for getter and setters to be defined with the same name. Only one needs to be changed to avoid name collision

```rust
#[getter]
fn x() {}

#[setter]
fn x() {} // op2 automatically converts this to __set_x to avoid name collision
```